moveit_ros/hybrid_planning/local_planner/local_planner_component/src/local_planner_component.cpp

```diff
@@ -119,11 +121,11 @@ bool LocalPlannerComponent::initialize()
     if (!trajectory_operator_instance_->initialize(node_ptr, planning_scene_monitor_->getRobotModel(),
                                                    "panda_arm"))  // TODO(sjahr) add default group param
```

Comment:
This TODO (and the other hardcoded `"panda_arm"` in this file) might be important to do before this is merged. It would be useful if you want to use hybrid planning on something besides the example panda moveit config